community/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,123 @@
+in the [culture](../culture) documentation pages. We strive to work together 
+as a community to realize these values.
+
+We have created this diversity statement because we believe that a diverse community is stronger, more vibrant, and more productive and motivated. A diverse community where people treat one another with respect has more potential contributors, more sources for ideas, and fewer shared assumptions that might hinder the achievement of our goals.
+
+Thus, we welcome all people to participate in SourceCred regardless of their identity
+or background.
+
+## Introduction & Scope
+
+This code of conduct should be honored by everyone who participates in the SourceCred community.
+It should be honored in any SourceCred activities including by not limited to
+online interactions (e.g. public and private mailing lists, issue trackers, discussion forums, wikis, and code repositories), meetups or conferences, and adhered to by anyone claiming affiliation 
+with the SourceCred community. This code of conduct is especially relevant for any
+group or individual that is representing the SourceCred community in an event
+(i.e., as speaker, volunteer, or other staff), and should govern governs standards of behaviors
+for the individual's participation in said event.
+
+This code of conduct is not exhaustive or complete, but rather serves to distill our understanding of a collaborative, inclusive community, and to provide avenues for seeking support if this
+code is breached. We should try to follow this code in spirit as much as in letter, to create a friendly and productive environment that enriches the SourceCred community.
+
+Unless explicitly specified for a particular project or initiative, this code
+of conduct serves as a global standard for all SourceCred affiliated projects, activities,
+and events. If an individual is found to violate this Code of Conduct, the Benevolent
+Dictator may recruit an advisory committee from the community and tkae any action they
+deem appropriate to resolve the situation including warning the person in question,
+removing them from a specific activity, or banning them from further participation in 
+activities (in person or online). The last option is unlikely, but possible in extreme
+circumstances.
+
+<br>
+
+## Standards for Behavior
+
+All communication should be appropriate for a professional audience including people of many different backgrounds. Please be yourself, but be respectful of others, and always be kind and courteous. There is never a need to be mean, rude, or disrespectful. Specifically, the SourceCred community strives to:
+
+ - **Be empathetic**. Realize that every post, event, project, or program is created by human beings that are likely trying their best, and aren't perfect. Try your best to understand perspectives that differ from your own, and to ask for clarification when you don't understand how someone feels.
```

Comment:
Love both of these sentences. I almost want to use them earlier in the doc, or otherwise highlight it more. It works as a good fist bullet point too though.

Reply:
Thanks @s-ben! How would you like to highlight it more, or what different changes would you suggest?

Reply:
Just re-read the latest draft and it all looks good!